diff --git a/tools/compile.bat b/tools/compile.bat
index 8741968..c9fcbe0 100644
--- a/tools/compile.bat
+++ b/tools/compile.bat
@@ -17,7 +17,7 @@ set OUT_PATH=".\out\%1"
 set CLS_PATH=".\third_party\jogamp\jar\*;.\third_party\junit\*;.\java"
 
 if not exist %OUT_PATH% mkdir %OUT_PATH%
-javac -Xdiags:verbose -cp %CLS_PATH% -d %OUT_PATH% java/tests/detailed/*.java java/tests/junittests/*.java java/tests/simple/*.java java/org/cef/*.java java/org/cef/browser/*.java java/org/cef/callback/*.java java/org/cef/handler/*.java java/org/cef/misc/*.java java/org/cef/network/*.java
+javac -cp %CLS_PATH% -d %OUT_PATH% java/tests/detailed/*.java java/tests/junittests/*.java java/tests/simple/*.java java/org/cef/*.java java/org/cef/browser/*.java java/org/cef/callback/*.java java/org/cef/handler/*.java java/org/cef/misc/*.java java/org/cef/network/*.java
 
 :: Copy resource files.
 xcopy /sfy .\java\tests\detailed\handler\*.html %OUT_PATH%\tests\detailed\handler\
